### PR TITLE
Image::place() fixed unexpected output with transparency image #56

### PR DESCRIPTION
**Pull request by romanmatyus**

### CODE REVIEW

**Comment by JanTvrdik** [comment]
Do we need the three `image_copy` calls even when `$opacity` is 100 (default and most common case)?

**Comment by romanmatyus** [comment]
Good point! I fixed it.

File: src/Utils/Image.php [hunk 1]
```diff
@@ -487,8 +487,18 @@ public function place(Image $image, $left = 0, $top = 0, $opacity = 100)
 			);
 
 		} elseif ($opacity <> 0) {
+			$cutting = imagecreatetruecolor($this->getWidth(), $this->getHeight());
```
Comment (JanTvrdik) [inline src/Utils/Image.php]: This should be `imagecreatetruecolor($image->getWidth(), $image->getHeight())`

File: src/Utils/Image.php [hunk 2]
```diff
@@ -487,8 +487,18 @@ public function place(Image $image, $left = 0, $top = 0, $opacity = 100)
...
+			imagecopy(
+				$cutting, $this->image,
+				0, 0, $left, $top, $this->getWidth(), $this->getHeight()
```
Comment (JanTvrdik) [inline src/Utils/Image.php]: This should be `0, 0, $left, $top, $image->getWidth(), $image->getHeight()`

**Comment by romanmatyus** [comment]
@JanTvrdik Thanks! Now it's many times faster.

**Comment by dg** [comment]
Thanks!

**Pull request merged**